app/controllers/cancelRegistration/CancelRegistrationCYAController.scala

```diff
@@ -38,32 +38,27 @@ class CancelRegistrationCYAController @Inject()(
                                             view: CancelRegistrationCYAView
                                           ) extends FrontendBaseController with I18nSupport with SummaryListFluency {
 
-  def onPageLoad(): Action[AnyContent] = controllerActions.withRequiredJourneyData(SelectChange.CancelRegistration) {
+  def onPageLoad(): Action[AnyContent] = controllerActions.withRequiredJourneyData(CancelRegistration) {
     implicit request =>
-
       (request.userAnswers.get(CancelRegistrationDatePage), request.userAnswers.get(ReasonPage)) match {
         case (Some(date: LocalDate), Some(reason: String)) =>
           val orgName: String = " " + request.subscription.orgName
-
-          val cancelRegistrationDateSummary: (String, SummaryList) = ("", SummaryListViewModel(
-            rows = Seq(CancelRegistrationDateSummary.row(request.userAnswers)))
+          val cancelRegistrationSummary: (String, SummaryList) = ("", SummaryListViewModel(
```

Comment:
Could we consider `@messages("cancelRegistration.checkYourAnswers.subheading")@orgName` to be the `String` part of `cancelRegistrationSummary` but with a non-bold font-weight CSS class on? I think this might be neater.

It feels like more of a h2 than a p to me intuitively. That being said, happy to be wrong on this but interested to see what you think?